modules/core/ui/dashboard/css/dashboard_layout.css

```diff
@@ -1,8 +1,40 @@
+/* Add a gap between layout regions. */
+.layout--twocol {
+  --dashboard-spacing: 1rem;
+  gap: var(--dashboard-spacing);
```

Comment:
This creates an implicit dependency on `farm_ui_theme` because that's the only place `--dashboard-spacing` is defined.

Can we move all of these dashboard style tweaks (including the ones that were here originally) to `modules/core/ui/theme/css/dashboard.css` and not worry about styling at all in the `farm_ui_dashboard` module?

Reply:
Actually `--dashboard-spacing` is defined here on line 3 as well. We're just overriding the variable in `farm_ui_theme`.

> Can we move all of these dashboard style tweaks (including the ones that were here originally) to modules/core/ui/theme/css/dashboard.css and not worry about styling at all in the farm_ui_dashboard module?

I suppose we could, it would be simpler

Reply:
Pushed new commits that first move the styles to `farm_ui_theme` and then add on gin related spacing/styling